Dockerfile

```
@@ -1,16 +1,27 @@
-#
-FROM python:3.11-slim
+FROM python:3.11-buster as builder
```

```suggestion
FROM python:3.11-buster AS builder
```

Отримав такий ворнінг у себе.
```
 2 warnings found (use --debug to expand):
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 18)
```